frontend/apps/reader/modules/readerlink.lua

```diff
@@ -99,14 +99,14 @@ local function isSwipeToGoBackEnabled()
     return G_reader_settings:isTrue("swipe_to_go_back")
 end
 
-local function isSwipeToFollowFirstLinkEnabled()
-    return G_reader_settings:isTrue("swipe_to_follow_first_link")
-end
-
 local function isSwipeToFollowNearestLinkEnabled()
     return G_reader_settings:isTrue("swipe_to_follow_nearest_link")
 end
 
+local function isSwipeIgnoreExternalLinksEnabled()
+    return G_reader_settings:nilOrTrue("swipe_ignore_external_links")
```

Comment:
```suggestion
    return G_reader_settings:isTrue("swipe_ignore_external_links")
```

Reply:
I disagree for swipe, which is there for a few years and evoke the physical gesture of turning pages to jump to footnotes and back. Having it suddently throw a popup is not a good thing.
(People like you who would still want that can now activate it - but no one else but you complained it was missing :)